### ISSUE TITLE
Low Accuracy When Reproducing Results Using train_mms_model.py with --use_video_ig65m and --use_image_vit

### ISSUE THREAD

**Issue by Koki05410** [Jason-Qiu/MultiSum_model#4]
I am trying to reproduce the results mentioned in the paper by running the following command with the published dataset in Google Drive:

`python3.8 train_mms_model.py --use_video_ig65m --use_image_vit
`

After training, I evaluated the the trained model with test dataset, setting
TEST_OR_VAL="test"
in test_mms_model.py.

Then I run
`python3.8 test_mms_model.py `

and finally I got the following results.
--------------------------------------------------------------------------------
DATALOADER:0 VALIDATE RESULTS
{'BERTSCORE_F1': 0.7763835191726685,
 'BERTSCORE_P': 0.779540479183197,
 'BERTSCORE_R': 0.7743812203407288,
 'BLEU': 0.7899070382118225,
 'CIDER': 0.005907426588237286,
 'F1_10_top': 0.17466329038143158,
 'F1_1_top': 0.3344867527484894,
 'F1_5_top': 0.28703710436820984,
 'Kendall_tau': nan,
 'METEOR': 0.0,
 'PSNR': 78.84628295898438,
 'Pearson_r': nan,
 'Pre_10_top': 0.09606481343507767,
 'Pre_1_top': 0.3333333432674408,
 'Pre_5_top': 0.17222225666046143,
 'RMSE': 0.00011448607256170362,
 'ROUGE_RAW_1_F': 4.519999980926514,
 'ROUGE_RAW_1_P': 8.779999732971191,
 'ROUGE_RAW_1_R': 4.03000020980835,
 'ROUGE_RAW_2_F': 0.3100000023841858,
 'ROUGE_RAW_2_P': 0.6600000262260437,
 'ROUGE_RAW_2_R': 0.23999999463558197,
 'ROUGE_RAW_L_F': 4.119999885559082,
 'ROUGE_RAW_L_P': 8.0,
 'ROUGE_RAW_L_R': 3.700000047683716,
 'Rec_10_threshold_0.75': 0.9606481194496155,
 'Rec_10_threshold_0.9': 0.9606481194496155,
 'Rec_10_top': 0.9606481194496155,
 'Rec_1_threshold_0.75': 0.33564814925193787,
 'Rec_1_threshold_0.9': 0.33564814925193787,
 'Rec_1_top': 0.33564814925193787,
 'Rec_5_threshold_0.75': 0.8611111044883728,
 'Rec_5_threshold_0.9': 0.8611111044883728,
 'Rec_5_top': 0.8611111044883728,
 'SPICE': 0.0,
 'SRE': -21.806026458740234,
 'SSIM': 0.9999827146530151,
 'cnn_cos_scores': 0.0091895442456007,
 'mAP_threshold_0.75': 0.5488203763961792,
 'mAP_threshold_0.9': 0.5488203763961792,
 'mAP_top': 0.5488203763961792}
--------------------------------------------------------------------------------

In addition, the trained model generated awful text summary.

```
>>> import numpy as np
>>> loaded_dic = np.load("results_whole/results_whole_0.npy", allow_pickle=True).item()
>>> loaded_dic["sentences"]
['Intro Cooking Foods', '<extra_id_0> Chickens', '<extra_id_0> Chickens', '<extra_id_0> Chickens', '<extra_id_0> Chicken', '<extra_id_0> <extra_id_31> Chickens', '<extra_id_0> <extra_id_53>sa Chicken', '<extra_id_0> Turkeys and the a Chicken', '<extra_id_0> Salmoned Eggs', '<extra_id_0> <extra_id_53> Chickens', '<extra_id_0> Salmoned Avocados', '<extra_id_0> Gingered Avocados', '<extra_id_0> Intro', '<extra_id_0> как Why a Chicken', '<extra_id_0> <extra_id_53> Chicken Caramel', '<extra_id_0> Gingers']
```

Questions:

Are there any additional steps or configurations needed to achieve the reported results?
Could there be an issue with the dataset or preprocessing steps?
Are there any known issues with the current version of the script that might affect model accuracy?

**Comment by lousiming** [comment]
> I am trying to reproduce the results mentioned in the paper by running the following command with the published dataset in Google Drive:
> 
> `python3.8 train_mms_model.py --use_video_ig65m --use_image_vit `
> 
> After training, I evaluated the the trained model with test dataset, setting TEST_OR_VAL="test" in test_mms_model.py.
> 
> Then I run `python3.8 test_mms_model.py `
> 
> ## and finally I got the following results.
> ## DATALOADER:0 VALIDATE RESULTS
> {'BERTSCORE_F1': 0.7763835191726685,
> 'BERTSCORE_P': 0.779540479183197,
> 'BERTSCORE_R': 0.7743812203407288,
> 'BLEU': 0.7899070382118225,
> 'CIDER': 0.005907426588237286,
> 'F1_10_top': 0.17466329038143158,
> 'F1_1_top': 0.3344867527484894,
> 'F1_5_top': 0.28703710436820984,
> 'Kendall_tau': nan,
> 'METEOR': 0.0,
> 'PSNR': 78.84628295898438,
> 'Pearson_r': nan,
> 'Pre_10_top': 0.09606481343507767,
> 'Pre_1_top': 0.3333333432674408,
> 'Pre_5_top': 0.17222225666046143,
> 'RMSE': 0.00011448607256170362,
> 'ROUGE_RAW_1_F': 4.519999980926514,
> 'ROUGE_RAW_1_P': 8.779999732971191,
> 'ROUGE_RAW_1_R': 4.03000020980835,
> 'ROUGE_RAW_2_F': 0.3100000023841858,
> 'ROUGE_RAW_2_P': 0.6600000262260437,
> 'ROUGE_RAW_2_R': 0.23999999463558197,
> 'ROUGE_RAW_L_F': 4.119999885559082,
> 'ROUGE_RAW_L_P': 8.0,
> 'ROUGE_RAW_L_R': 3.700000047683716,
> 'Rec_10_threshold_0.75': 0.9606481194496155,
> 'Rec_10_threshold_0.9': 0.9606481194496155,
> 'Rec_10_top': 0.9606481194496155,
> 'Rec_1_threshold_0.75': 0.33564814925193787,
> 'Rec_1_threshold_0.9': 0.33564814925193787,
> 'Rec_1_top': 0.33564814925193787,
> 'Rec_5_threshold_0.75': 0.8611111044883728,
> 'Rec_5_threshold_0.9': 0.8611111044883728,
> 'Rec_5_top': 0.8611111044883728,
> 'SPICE': 0.0,
> 'SRE': -21.806026458740234,
> 'SSIM': 0.9999827146530151,
> 'cnn_cos_scores': 0.0091895442456007,
> 'mAP_threshold_0.75': 0.5488203763961792,
> 'mAP_threshold_0.9': 0.5488203763961792,
> 'mAP_top': 0.5488203763961792}
> In addition, the trained model generated awful text summary.
> 
> ```
> >>> import numpy as np
> >>> loaded_dic = np.load("results_whole/results_whole_0.npy", allow_pickle=True).item()
> >>> loaded_dic["sentences"]
> ['Intro Cooking Foods', '<extra_id_0> Chickens', '<extra_id_0> Chickens', '<extra_id_0> Chickens', '<extra_id_0> Chicken', '<extra_id_0> <extra_id_31> Chickens', '<extra_id_0> <extra_id_53>sa Chicken', '<extra_id_0> Turkeys and the a Chicken', '<extra_id_0> Salmoned Eggs', '<extra_id_0> <extra_id_53> Chickens', '<extra_id_0> Salmoned Avocados', '<extra_id_0> Gingered Avocados', '<extra_id_0> Intro', '<extra_id_0> как Why a Chicken', '<extra_id_0> <extra_id_53> Chicken Caramel', '<extra_id_0> Gingers']
> ```
> 
> Questions:
> 
> Are there any additional steps or configurations needed to achieve the reported results? Could there be an issue with the dataset or preprocessing steps? Are there any known issues with the current version of the script that might affect model accuracy?

Could you let me know if you downloaded the original video? If you did, could you please send me a message? It would be very helpful for me, and I hope you can assist me with this.

**Comment by wsjqkd** [comment]
Did you obtain the original video?
